docs/next.config.js

```diff
@@ -72,7 +72,7 @@ module.exports = withNextPluginPreval({
       },
       {
         source: '/ui/theme/alternativeStyling',
-        destination: '/theming/alternative-styling',
```

Comment:
I think we also need a redirect from `/theming/alternative-styling` to new url `/guides/css-in-js'

Reply:
Good catch! Done